scrapy/pipelines/images.py

```diff
@@ -46,7 +46,7 @@ class ImagesPipeline(FilesPipeline):
     THUMBS = {}
     DEFAULT_IMAGES_URLS_FIELD = 'image_urls'
     DEFAULT_IMAGES_RESULT_FIELD = 'images'
-    IMAGES_S3_SECURE_CONNECTION = False
+    images_s3_secure_connection = False
```

Comment:
@redapple , hmm, yes, this is true, what should be done with images_s3_secure_connection for ImagesPipeline?